nilearn/conftest.py

```diff
@@ -315,3 +316,21 @@ def testdata_4d_for_plotting():
         "img_atlas": img_atlas,
         "atlas_labels": atlas_labels,
     }
+
+
+@pytest.fixture
+def set_username():
+    """Set the username for the duration of the test."""
+    try:
+        username = getpass.getuser()
+    except ModuleNotFoundError:
+        username = None
+
+    os.environ["USER"] = "nilearn"
+
+    yield
+
+    if username is None:
+        del os.environ["USER"]
+    else:
+        os.environ["USER"] = username
```

Comment:
Just to be clear, is the error that warrants this related to this PR? If so, can you explain?
Haven't seen it on main or in other PRs. More just for my understanding...

Reply:
not seen in any other PR either

so far I was trying to solve this to at least make sure that my assumptions about what the problem is

Reply:
Seems related to this: https://github.com/tox-dev/tox/issues/1455

May be easier to solve in the tox env definition: let me check.

Reply:
ok so you can pass env variables in the tox config and this solves the problem: good thing you pushed me to look further